programs/zstdcli.c

```diff
@@ -716,10 +716,10 @@ int main(int argCount, const char* argv[])
     /* Welcome message (if verbose) */
     DISPLAYLEVEL(3, WELCOME_MESSAGE);
 
-    if (nbThreads == 0) {
-        /* try to guess */
-        nbThreads = UTIL_countPhysicalCores();
-        DISPLAYLEVEL(3, "Note: %d physical core(s) detected \n", nbThreads);
+    if (nbWorkers == 0) {
```

Comment:
It looks like there isn't any way to disable async mode, given that zstd is compiled with threading support. Do we want to provide a way to do that or not?

Reply:
That's a good point.
I'm not too sure if "disabling async mode" is a useful feature though.
I fail to find a good use case to justify another command.

Reply:
Actually, there is a good enough reason : memory usage.
The single-thread mode currently uses much less memory than `zstdmt`.
So that could be an argument.

Reply:
Still looking for a good name for such command.
I looked into `pigz` and `pxz` command list, just in case there would be something similar, but found nothing.

So now, inventing yet another command, it could be : 
- `--lowmem` : to emphasize memory usage reduction
- `--single-thread` : as the name implies
- `--synchronous` : to make it opposed to "asynchronous"

The second option looks the clearer one.

Reply:
I like `--single-thread`.